backend/services/application-plane/problem-service/.env.example

```
@@ -0,0 +1,8 @@
+PORT=3100
+DYNAMODB_TABLE=TenkaCloud-dev
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DYNAMODB_TABLE key should go before the PORT key

(UnorderedKey)